docker/images/alpine/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM alpine:3.8
+MAINTAINER Postman Labs <help@getpostman.com>
+
+ARG NODE_VERSION=8
+ARG NEWMAN_VERSION
+
+# Set environment variables
+ENV LC_ALL="en_US.UTF-8" LANG="en_US.UTF-8" LANGUAGE="en_US.UTF-8" ALPINE_NODE_REPO="oznu/alpine-node"
+
+# Bail out early if NODE_VERSION is not provided
+RUN if [ ! $(echo $NEWMAN_VERSION | grep -oE "^[0-9]+\.[0-9]+\.[0-9]+$") ]; then \
+        echo "\033[0;31mA valid semver Newman version is required in the NEWMAN_VERSION build-arg\033[0m"; \
+        exit 1; \
+    fi && \
+    # Add the latest Alpine Linux package to the repositories list
+    echo -e "http://nl.alpinelinux.org/alpine/v3.8/main\nhttp://nl.alpinelinux.org/alpine/v3.8/community" > /etc/apk/repositories && \
```

Comment:
Is manually adding these two repositories mandatory? I removed this line and the image seemed to work just fine (using the test requests from https://www.getpostman.com/collections/8a0c9bc08f062d12dcda).

Reply:
@namelessvoid This is necessary for Alpine 3.3, we should remove them if the image builds and runs fine 😄

Reply:
@namelessvoid 
**Update:** I tried building this locally, and there were consistent errors installing dependencies. This could be due to our builds picking up from different cache edge locations. Let's stick with this for consistency.